### ISSUE TITLE
duplicate activities coming through to home kit

### ISSUE THREAD

**Issue by gigafonzie** [KraigM/homebridge-harmonyhub#66]
Hi guys, hoping perhaps an easy one!

Im getting the activities come through twice from homebridge. Only the first set works though, the second seems to be a dud. What can i do to get rid of these 'ghost' switches?

Thanks

Ben

**Comment by tomto147** [comment]
Try deleting user/.homebridge/persist and user/.homebridge/accessories catalogues and rerun the homebridge

**Comment by gigafonzie** [comment]
yep that got it! deleted /var/homebridge/persist and accessories.

Thanks very much!
